setup.py

```diff
@@ -1,9 +1,27 @@
 from setuptools import setup, find_packages
 from codecs import open
-from os import path
+from os import path, environ
+from json import load
 
 DESCRIPTION = 'Python library for SPIFFE'
 
+# https://stackoverflow.com/questions/49496994/how-do-i-sync-values-in-setup-py-install-requires-with-pipfile-packages
+def locked_requirements(section):
+    # Explicitly adding an absolute path to PWD looks odd, but there's some
+    # directory shenangians that happen without it
+    with open(path.join(environ['PWD'], 'Pipfile.lock')) as pip_file:
+        pipfile_json = load(pip_file)
+
+    if section not in pipfile_json:
+        print("{0} section missing from Pipfile.lock".format(section))
```

Comment:
Can you update this line to use single quotes please?

Reply:
I can - that whole thing is copy/pasted from the link in the source.